documentation/library_guide/onedpl_gsg.rst

```diff
@@ -47,8 +47,8 @@ Usage Examples
 `oneAPI GitHub samples repository <https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneDPL>`_.
 Each sample includes a readme with build instructions.
 
-oneapi/dpl/random Usage Example
--------------------------------
+``<oneapi/dpl/random>`` Usage Example

```

Comment:
This file doesn't related to necessary change but I cannot just miss that since it looked ugly.

Reply:
Seems nobody objects. Great.

Reply:
To avoid code but make it less ugly I suggest, "oneapi/dpl/random Header Usage Example".

Reply:
And what is the problem with the code in the title? Is it some restriction according to guidelines or is it because we previously didn't do that?

Reply:
Done, but it would be great if you share some document with guidelines for documentation writing. It is also related to other comments like "We don't use Italics", "We don't use quotes", etc.